lib/ansible/modules/iptables.py

```diff
@@ -268,8 +268,8 @@
     type: str
   ctstate:
     description:
-      - C(ctstate) is a list of the connection states to match in the conntrack module.
-      - Possible states are C(INVALID), C(NEW), C(ESTABLISHED), C(RELATED), C(UNTRACKED), C(SNAT), C(DNAT)
+      - I(ctstate) is a list of the connection states to match in the conntrack module.
```

Comment:
```suggestion
      - A list of the connection states to match in the conntrack module.
```

Reply:
I used automation to find those fixed things, so didn't focus on other things but this will make the doc much better. Thanks!